Chatkit/Authentication/Token Providers/TestTokenProvider.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+import PusherPlatform
+
+/// TestTokenProvider retrieves tokens from the Chatkit service's Test Token Provider, which
+/// is for development user only, and must be enabled for your instance in the Chatkit Dashboard.
```

Comment:
Sorry didn't spot this earlier, should this be "development use" rather than "development use**r**"?

Reply:
👍